autosklearn/automl.py

```diff
@@ -110,7 +111,8 @@ def __init__(self,
                  exclude_preprocessors=None,
                  resampling_strategy='holdout-iterative-fit',
                  resampling_strategy_arguments=None,
-                 shared_mode=False,
+                 n_jobs=None,
```

Comment:
As shared mode is going away, we can also remove this line (from below, I found it on a grep on a repo)?
https://github.com/automl/auto-sklearn/blob/00ecf0783512e95c46459f7e4a084ea805cffb39/test/test_automl/test_automl.py#L44